src/components/QuickOpenModal.js

```diff
@@ -41,6 +43,7 @@ type Props = {
   sources: Array<Object>,
   selectedSource?: SourceRecord,
   query: string,
+  generalItems: Array<Object>,
```

Comment:
I don't think this prop is ever used? Also wouldn't it be `FormattedGeneralQuery[]`?